interrogatio/widgets/wizard.py

```diff
@@ -297,7 +300,11 @@ def next(self):
     def validate(self):
         step = self.steps[self.current_step_idx]
         handler = step['handler']
-        if handler and not handler.is_valid(self.answers) and not handler.is_disabled():
+        if (
+                handler
+                and not handler.is_valid(self.answers)
+                and not handler.is_disabled()
```

Comment:
why not `handler.is_disabled(self.answers)` ?
IMO context should be mandatory for is_disabled.

Reply:
It is a typo. For sure context is mandatory